medical/data/medical_tour.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Make the xmlid of menus required by the Medical tour available in webclient -->
+    <record id="medical.medical_root" model="ir.ui.menu">
```

Comment:
It technically is redundant with https://github.com/laslabs/vertical-medical/pull/37/files#diff-51a3344074644d0761ab5ff7e70d0b85R7

The issue is that `load_xmlid` is not a valid attribute when using the menu tag, so it has to be appended after the actual menu creation. This is the same pattern Odoo SA uses for their tours, even when the menu is located in the same module, so it seems there's not a way around this.